code/game/objects/machinery/doors/door.dm

```diff
@@ -29,6 +31,8 @@
 	var/list/fillers
 	//used for determining emergency access
 	var/emergency = FALSE
+	var/cyclelinkeddir = 0
+	var/obj/machinery/door/airlock/cycle_linked_airlock
```

Comment:
I won't make you autodoc all the old vars, but you need to atleast autodoc the stuff you add.